src/main/java/fi/tkgwf/ruuvi/common/bean/RuuviMeasurement.java

```diff
@@ -212,6 +248,165 @@ public void setMeasurementSequenceNumber(Integer measurementSequenceNumber) {
+    public Integer getVocIndex() {
+        return vocIndex;
+    }
+
+    public void setVocIndex(Integer vocIndex) {
+        this.vocIndex = vocIndex;
+    }
+
+    /**
+     * Get NOx index, unitless.
+     * <p>
+     * For details, see https://docs.ruuvi.com/communication/bluetooth-advertisements/data-format-6
+     *
+     * @return measurement or null if not available
+     */
+    public Integer getNoxIndex() {
+        return noxIndex;
+    }
+
+    public void setNoxIndex(Integer noxIndex) {
+        this.noxIndex = noxIndex;
+    }
+
+    /**
+     * Get luminosity, in lux.
+     * <p>
+     * For details, see https://docs.ruuvi.com/communication/bluetooth-advertisements/data-format-6
+     *
+     * @return measurement or null if not available
+     */
+    public Double getLuminosity() {
+        return luminosity;
+    }
+
+    public void setLuminosity(Double luminosity) {
+        this.luminosity = luminosity;
+    }
+
+    /**
+     * Get PM 1.0, in micrograms per cubic meter.
+     * <p>
+     * For details, see https://docs.ruuvi.com/communication/bluetooth-advertisements/data-format-6
+     *
+     * @return measurement or null if not available
+     */
+    public Double getPm1() {
+        return pm1;
+    }
+
+    public void setPm1(Double pm1) {
+        this.pm1 = pm1;
+    }
+
+    /**
+     * Get PM 4.0, in micrograms per cubic meter.
+     * <p>
+     * For details, see https://docs.ruuvi.com/communication/bluetooth-advertisements/data-format-6
+     *
+     * @return measurement or null if not available
+     */
+    public Double getPm4() {
+        return pm4;
+    }
+
+    public void setPm4(Double pm4) {
+        this.pm4 = pm4;
+    }
+
+    /**
+     * Get PM 10.0, in micrograms per cubic meter.
+     * <p>
+     * For details, see https://docs.ruuvi.com/communication/bluetooth-advertisements/data-format-6
+     *
+     * @return measurement or null if not available
+     */
+    public Double getPm10() {
+        return pm10;
+    }
+
+    public void setPm10(Double pm10) {
+        this.pm10 = pm10;
+    }
+
+    /**
+     * Get sensor calibration in progress flag.
+     * <p>
+     * For details, see https://docs.ruuvi.com/communication/bluetooth-advertisements/data-format-6#flags
+     *
+     * @return true if calibration is in progress (values still improving), false if complete, null if not available
+     */
+    public Boolean isCalibrationInProgress() {
+        return calibrationInProgress;
+    }
+
+    public void setCalibrationInProgress(Boolean calibrationInProgress) {
+        this.calibrationInProgress = calibrationInProgress;
+    }
+
+    /**
+     * Calculate Air Quality Index from PM2.5 and CO2 measurements.
+     * <p>
+     * For algorithm details, see https://github.com/ruuvi/com.ruuvi.station.webui/blob/master/src/decoder/untils.js
+     *
+     * @return Air Quality Index (0-100, higher = better) or null if not available
+     */
+    public Double calculateAirQualityIndex() {
```

Comment:
This doesn't really belong here as it's not transmitted by Air. It would be more appropriate in RuuviCollector's [MeasurementValueCalculator](https://github.com/Scrin/RuuviCollector/blob/master/src/main/java/fi/tkgwf/ruuvi/utils/MeasurementValueCalculator.java) (which could be moved here in its entirety though)

Reply:
Now done in https://github.com/Scrin/ruuvitag-common-java/pull/6/commits/a279f04457bea95c96d41ecd83fb61860ae81671. Regression tests also included in https://github.com/Scrin/ruuvitag-common-java/pull/6/commits/37a4c4b4bba6eab7ad3e401a7f33f274d88a635d